content/articles/how-sweet-is-kubernetes/index.md

```diff
@@ -0,0 +1,254 @@
+
+**WHAT KUBERNETES IS NOT**
+
+Kubernetes is not an alternative to Docker. It is something more.
+
+**WHAT KUBERNETES IS**
+
+In proper definition: Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications.
+
+**KUBERNETES Vs. DOCKER**
+
+| s/n | DOCKER  |  KUBERNETES  |
+|----|----------------------------|:----------------------------------    |
+| 1.|Docker is used for building and contanerizing images.   |Those container built, the function of kubernetes is to manage them effectively, hence you can say Kubernetes is an orchestration tool.  |
+| 2. |Docker runs on a single node. | Kubernetes runs across clusters. |
+| 3. |Docker is built by three y-connector graduates. | Kubernetes is built by google engineers before been made an open source project. |
+
+However, it’s better to see for ourselves how it works with the aid of the Fleetman application that I deployed on my repo as the instructional material.
+
+**MAJOR CONCEPT IN KUBERNETES**
+
+**DEPLOYMENT**
+
+Deployment is a configuration that provides declarative updates to applications. It allows you to describe an application's life cycle, such as which images to use for the app, the number of pods there should be, and the way they should be updated. It is also a configuration where you can determine the number of replica sets to be produced per pod. It helps maintain the cluster in order to avoid the minutest downtime. We can say it is like a blueprint that dictates how every pod feature will behave.
+
+The kind of configuration we are declaring is deployment while, the name is position simulator. The container we used is on line 17, the position simulator release 1 from Richard Chesterwood repository. As we can see, ‘replicas’ is being specified as our replica set, and the template is what declares our pod. For a standard, YAML file even though can be separated, to ensure orderliness and separation of concerns, we write the services of each deployment in the same file, separating them with three dashes(-). Some do arrange theirs as all services in a separate file, that too is acceptable.
+
+```
```

Comment:
So he it comes to displaying any example code block we normally write which language is being used e.g
I assume this is yaml. So you should have it as  
``` yaml
insert code in here
````
If it's maybe java I'll have it as :
```java
Insert code here
````
So make this change to all your code blocks.

Reply:
OK, check for corrections